src/api/apiFetch.js

```diff
@@ -41,27 +45,14 @@ export const apiCall = async (
   try {
     networkActivityStart(isSilent);
     const response = await apiFetch(auth, route, params);
-
-    if (!response.ok) {
-      console.log('Bad response for:', { auth, route, params, response }); // eslint-disable-line
-      const error = new Error('API');
-      // $FlowFixMe
-      error.response = response;
-      throw error;
+    if (response.status === 401) {
+      // UNAUTHORIZED
+      // apiKey got exprired
+      // dispatch logout action
+      store.dispatch(logout());
+      showToast('Please login again');
```

Comment:
@roberthoenig This is a fix for #2770. It will be great if could test & verify fix.

cc @borisyankov 

(I am simplified this from #2118, which is kinda too big)

Reply:
I don't think we need the toast message.
My concern is I don't like `apiFetch` depending on Redux, knowing even about it.
On the other hand, this solution is super simple and not bad, and the alternatives I am having in my head seem too convoluted.